roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```diff
@@ -0,0 +1,48 @@
+- name: Verify mariadb scrapeconfig exists
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-xxxx
+    common_cr_list:
+      - kind: scrapeconfigs.monitoring.rhobs
+        name: telemetry-mysqld-exporter
+
+- name: Get galera CR names
+  ansible.builtin.shell: |
+    oc get galeras -o custom-columns=NAME:.metadata.name --no-headers
+  register: cr_names
+  changed_when: false
+  failed_when: cr_names.rc >= 1 or cr_names.stdout == ""
+
+- name: Create mariadb pod list
+  ansible.builtin.set_fact:
+    common_pod_list: "{{ common_pod_list | default([]) + [item + '-galera-0' + '-mysqld-exporter-0'] }}"
+  loop: "{{ cr_names.stdout_lines }}"
+
+- name: Verify galera pods are running
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_pod_test_id: RHOSO-xxxx
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
+
+- name: Check mariadb metric endpoints
```

Comment:
This is strange. Looking at the must-gather, I'm pretty sure this task should fail, but it was successful. I'll take a further look tomorrow.

Reply:
I can't replicate this behavior locally. Maybe the k8s serivice was actually there, so the command worked because of that?